script.js

```diff
@@ -57,49 +84,99 @@ const addCardContainer = () => {
 
   const inputField = document.createElement('input');
   inputField.setAttribute("type", "text");
-  inputField.setAttribute("id", id);
   inputField.setAttribute("placeholder", "Add something...");
-  inputField.addEventListener("keydown", (e) => {
-    if (inputField.value) {
-      if (e.key === "Enter") {
-        addListItem();
-      } else if (e.key === "Escape") {
-        inputField.value = "";
-      }
-      saveData();
-    };
-  });
 
   const listButton = document.createElement('button');
   listButton.setAttribute("class", "button");
   listButton.innerHTML = "Add";
-  listButton.addEventListener("click", () => {
-    if (inputField.value) {
-      addListItem();
-    };
-  });
 
   const listContainer = document.createElement('ul');
   listContainer.setAttribute("class", "list-container");
 
-  container.appendChild(cardContainer);
-
   cardContainer.append(deleteCardIcon, cardTitle, inputContainer, listContainer);
-  
+
   inputContainer.append(inputField, listButton);
 
-  listContainer.addEventListener("click", (e) => {
-    if (e.target.tagName.toLowerCase() === deleteItemTag) {
-      e.target.parentElement.remove();
-    } else if (e.target.tagName.toLowerCase() === inputItemTag) {
-      e.target.toggleAttribute('checked');
+  const todoListItemElems = todoListState
+    .map((todoListItemData) => renderTodoListItem(todoListItemData))
```

Comment:
`renderTodoListItem(todoListItemData, ???)` - missed the second argument

Reply:
Added)